internal/cephfs/volume.go

```diff
@@ -135,20 +138,16 @@ func createVolume(ctx context.Context, volOptions *volumeOptions, cr *util.Crede
 		clusterAdditionalInfo[volOptions.ClusterID] = &localClusterState{}
 	}
 
+	ca, err := volOptions.conn.GetFSAdmin()
+	if err != nil {
+		util.ErrorLog(ctx, "failed to create volume %s: %s", volOptions.FsName, err)
```

Comment:
Need to change the logging?

Reply:
yeah, needs more reviewing all over. This is a draft so that others can see the general direction things are going